docs/src/main/asciidoc/_admin-guide/Galleon_provisioning.adoc

```diff
@@ -122,6 +122,7 @@ Basic layers:
 * _jaxrs_: Support for JAXRS. Depends on _web-server_ (defined in the WildFly servlet feature-pack).
 * _jms-activemq_: Support for connections to a remote JMS broker.
 * _jpa_: Support for JPA (using the latest WildFly supported Hibernate release).
+* _microprofile_: Support for MicroProfile specification including Config, Fault Tolerance, JWT, Metrics, OpenAPI, OpenTracing.
```

Comment:
I'm not sure I would bother enumerating the individual specs, unless we want to keep this list updated (e.g. health) with every new release.

Reply:
My thinking was to keep the list updated so people dont assume this includes some specs we dont yet support or ones that might be only incubating. What better wording do you suggest?

Reply:
Would including an additional qualifier in the name make sense?  Either platform version  or allow for a second one `microprofile_experimental`

Reply:
Reworded to be more general.